.github/workflows/ci.yml

```diff
@@ -455,3 +455,93 @@ jobs:
+    runs-on: macos-10.15
+    timeout-minutes: 30
+    needs: [project, linters, protos, man]
+
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13.15'
+
+      - name: Set env
+        shell: bash
+        run: |
+          echo "::set-env name=GOPATH::${{ github.workspace }}"
+          echo "::add-path::${{ github.workspace }}/src/github.com/containerd/containerd/bin"
+
+      - name: Checkout containerd
+        uses: actions/checkout@v2
+        with:
+          path: src/github.com/containerd/containerd
+      - name: Checkout runu
+        uses: actions/checkout@v2
+        with:
+          repository: ukontainer/runu
+          path: src/github.com/ukontainer/runu
+
+      - name: Install Build Deps
+        shell: bash
+        run: |
+          cd src/github.com/containerd/containerd
+          script/setup/install-dev-tools
+      - name: Install runtime
+        shell: bash
+        run: |
+          SHIM_COMMIT=v0.8.2
+          GO111MODULE=on go get -v github.com/ukontainer/runu@"${SHIM_COMMIT}"
+          bindir="$(pwd)/bin/"
+          cd src/github.com/ukontainer/runu
+          git fetch --tags origin "${SHIM_COMMIT}"
+          git checkout "${SHIM_COMMIT}"
+          GO111MODULE=on go build -o "${bindir}/containerd-shim-darwin-v1" ./cmd/containerd-shim-darwin-v1
+      - name: Binaries
+        shell: bash
+        run: |
+          set -o xtrace
+          cd src/github.com/containerd/containerd
+          make binaries
+      - name: Install containerd
+        run: |
+          make binaries
+          sudo make install
+        working-directory: src/github.com/containerd/containerd
+      - name: make test
+        env:
+          TEST_RUNTIME: "io.containerd.darwin.v1"
+        shell: bash
+        run: |
+          cd src/github.com/containerd/containerd
+          make test
+      - name: Integration 1
+        env:
+          TEST_RUNTIME: "io.containerd.darwin.v1"
+        shell: bash
+        run: |
+          cd src/github.com/containerd/containerd
+          WORKDIR="$(mktemp -d)"
+          # /run is read-only under macOS
+          export EXTRA_TESTFLAGS="-address=$WORKDIR/c1.sock"
+          export PATH=$PATH:$(go env GOPATH)/bin
+          # default GOPATH under sudo is $HOME/go, not $HOME/.go on macos
+          sudo -E make integration
+      # Run the integration suite a second time. See discussion in github.com/containerd/containerd/pull/175
+      - name: Integration 2
+        env:
+          TEST_RUNTIME: "io.containerd.darwin.v1"
+        shell: bash
+        run: |
+          cd src/github.com/containerd/containerd
+          WORKDIR="$(mktemp -d)"
+          rm -rf /var/run/containerd-test
+          rm -rf /var/lib/containerd-test
+          # /run is read-only under macOS
+          export EXTRA_TESTFLAGS="-address=$WORKDIR/c1.sock"
+          export TESTFLAGS_PARALLEL=1
+          export PATH=$PATH:$(go env GOPATH)/bin
```

Comment:
This can be simplified:
`export PATH` is not needed, since you already setup PATH from `Set env` above.
Same for `Integration 1`.

Reply:
fixed, thanks.